BREAKING_CHANGES.md

```diff
@@ -651,4 +651,21 @@ When running Jest directly, previously most of Jest had to be manually configure
 Stencil v3.0.0 is in development at this time and has not been released. The list below is not final and is subject to
 change. Further details on each of these items will be included prior to the release.
 
-- [fix(testing): puppeteer v10 support #2934](https://github.com/ionic-team/stencil/pull/2934)
\ No newline at end of file
+- [fix(testing): puppeteer v10 support #2934](https://github.com/ionic-team/stencil/pull/2934)
+
+## DEPRECATIONS
+
+### backwards compatibility for `@Component.assetsDir` EOL
+
+The `assetsDir` component prop was [#componentassetsdir](previously deprecated)
```

Comment:
I think these need to switched, the URL/anchor should occur in the parenthesis:
```suggestion
The `assetsDir` component prop was [previously deprecated](#componentassetsdir)
```

Reply:
haha ok not quite enough ☕ yet today